Resources/Prototypes/_RMC14/Entities/Mobs/Xeno/lesser_drone.yml

```diff
@@ -72,6 +72,7 @@
       - XenoParasite
       - XenoEgg
       - XenoFruit
+  - type: TunnelRenamer
```

Comment:
Is the lesser also supposed to be able to do this?

Reply:
Yup
https://github.com/cmss13-devs/cmss13/blob/570fc8f2777381f4a88b4029da2540c72f91b253/code/modules/mob/living/carbon/xenomorph/castes/lesser_drone.dm#L74